src/analysis/calculus/deriv.lean

```diff
@@ -56,6 +57,7 @@ We also show the existence and compute the derivatives of:
   - negation
   - subtraction
   - multiplication
+  - `star`
```

Comment:
to match the others
```suggestion
  - star
```
or 
```sugggestion
  - `x ↦ star x`
```

Reply:
```suggestion
  - conjugation `x ↦ star x`
```

Reply:
"conjugation" is too specific a word, right? This works for matrices over `ℝ` where the `star` operation is the adjoint.

Reply:
It doesn't, because matrices aren't normed. And it if did, we'd want a special lemma for non-square matrices anyway.